src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,41 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.CinemaHallRequestDto;
+import mate.academy.spring.dto.CinemaHallResponseDto;
+import mate.academy.spring.mapper.CinemaHallMapper;
+import mate.academy.spring.service.CinemaHallService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-halls")
+public class CinemaHallController {
+    private CinemaHallService cinemaHallService;
+    private CinemaHallMapper cinemaHallMapper;
+
+    public CinemaHallController(CinemaHallService cinemaHallService,
+                                CinemaHallMapper cinemaHallMapper) {
+        this.cinemaHallService = cinemaHallService;
+        this.cinemaHallMapper = cinemaHallMapper;
+    }
+
+    @GetMapping("/")
```

Comment:
mapping should be the same 
```suggestion
    @GetMapping
```